modules/data-store/src/data-store.js

```diff
@@ -0,0 +1,147 @@
+
+import serializer from "abitbol-serializable/lib/serializer";
+import minimatch from "minimatch";
+
+
+/**
+ * Stores project's entities and blobs.
+ */
+class DataStore {
+
+    constructor() {
+        this.entitiesByPath = {};
+        this.entitiesByUuid = {};
+    }
+
+    /**
+     * Add an entity to the store.
+     *
+     * @param {SerializableClass} entity The entity
```

Comment:
As you created a class called `Entity`, you should update the doc to replace `SerializableClass` by `Entity` :)

(There is other places in the file where this should be changed)

Reply:
Done